ENIGMAsystem/SHELL/Platforms/SDL/Window.cpp

```diff
@@ -32,6 +31,19 @@ namespace enigma {
 void (*WindowResizedCallback)();
 
 SDL_Window* windowHandle = nullptr;
+unsigned sdl_window_flags = SDL_WINDOW_HIDDEN;
+
+void init_sdl_window_attributes();
+
+bool initGameWindow() {
+  SDL_Init(SDL_INIT_VIDEO);
+  if (isSizeable) sdl_window_flags |= SDL_WINDOW_RESIZABLE;
```

Comment:
If we were organized, we'd have standardized these settings in an actual document, and would link the section about this in that document from here and everywhere else that implements these features to make it easier to make changes.

Reply:
Well, it seems as though you are trying to influence `platforms_mandatory.h` to be said document. However, that's fine, I totally agree.
https://github.com/enigma-dev/enigma-dev/blob/d0487e73f36b51de9f9f9fbbf4d62d73fa4a9043/ENIGMAsystem/SHELL/Platforms/platforms_mandatory.h#L72